routing.go

```diff
@@ -35,7 +37,7 @@ var asyncQueryBuffer = 10
 
 // PutValue adds value corresponding to given Key.
 // This is the top level "Store" operation of the DHT
-func (dht *IpfsDHT) PutValue(ctx context.Context, key string, value []byte) (err error) {
+func (dht *IpfsDHT) PutValue(ctx context.Context, key string, value []byte, options ...ropts.Option) (err error) {
```

Comment:
we should probably have a quorum option here too.

Reply:
We'd probably want to call it something different (different purpose) but yes, I agree.